src/Framework/test/TargetingPackTests.cs

```diff
@@ -20,17 +20,24 @@ public class TargetingPackTests
         private readonly string _expectedRid;
         private readonly string _targetingPackRoot;
         private readonly ITestOutputHelper _output;
+        private readonly bool _isTargetingPackBuilding;
 
         public TargetingPackTests(ITestOutputHelper output)
         {
             _output = output;
             _expectedRid = TestData.GetSharedFxRuntimeIdentifier();
             _targetingPackRoot = Path.Combine(TestData.GetTestDataValue("TargetingPackLayoutRoot"), "packs", "Microsoft.AspNetCore.App.Ref", TestData.GetTestDataValue("TargetingPackVersion"));
+            _isTargetingPackBuilding = bool.Parse(TestData.GetTestDataValue("IsTargetingPackBuilding"));
         }
 
         [Fact(Skip="https://github.com/aspnet/AspNetCore/issues/14832")]
```

Comment:
Should also unskip these 2

Reply:
Ah, I rebased from master, which didn't have these skipped.